scripts/secnetperf-helpers.psm1

```diff
@@ -483,6 +489,9 @@ function Invoke-Secnetperf {
         continue
     }
 
+     # Linux XDP requires sudo for now
+    $sudo = (!$IsWindows -and $io -eq "xdp") ? "sudo -E " : ""
```

Comment:
Can you explain why the `-E` needs to be here? As I understand it, it gives edit permissions? Is that for the output artifacts?

Reply:
sudo reset environment variables.
If my understanding is correct, LD_LIBRARY_PATH is not propagated to sudo execution.